Bio/PDB/Structure.py

```diff
@@ -7,37 +7,42 @@
 
 """The structure class, representing a macromolecular structure."""
 
+from typing import Generator
 from Bio.PDB.Entity import Entity
 from Bio.PDB.internal_coords import IC_Chain
+from Bio.PDB.Atom import Atom
```

Comment:
One of the nicer things of having a `Structure` and a `StructureBuilder` is that there isn't a strict dependency between what children `Structure` can have. Adding types adds all these imports that don't hurt much but would otherwise be completely unnecessary.  Also, if in the future we add another Entity subtype, we'd have to add it here to keep type hints working.

Is there a better way of handling this?